src/contracts/libraries/GPv2SafeERC20.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * @title GPv2SafeERC20
+ * @author Gnosis Developers
+ * @dev Gas-efficient version of Openzeppelin's SafeERC20 contract that does not
+ * revert when calling a non-contract.
+ */
+library GPv2SafeERC20 {
+    /**
+     * @dev Wrapper around a call to the ERC20 function `transfer` that reverts
+     * also when the token returns `false`.
+     */
+    function safeTransfer(
+        IERC20 token,
+        address to,
+        uint256 value
+    ) internal {
+        _callOptionalReturn(
+            token,
+            abi.encodeWithSelector(token.transfer.selector, to, value)
```

Comment:
I guess there is no nice way around this extra allocation per transfer. It doesn't look like it affects the benchmarks in a significant way so I don't think its a problem (and it was like that before as well).